MANIFEST.in

```diff
@@ -1,5 +1,15 @@
-include *.rst *.txt
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+include *.rst
+include *.txt
+include buildout.cfg
 include tox.ini
-graft src
-global-exclude *.pyc
-global-exclude *.pyo
```

Comment:
Isn't it much easier to use `graft src` instead of having to explicitly specify every suffix with `recursive-include`, with the danger of forgetting to edit this manifest when you add a file with an unknown suffix, leading to a broken release unless you use `check-manifest`?
I don't really mind it for this package, but it may be something for the meta project. But I can imagine this has been discussed already, and that we prefer explicit `recursive-include` over implicit `graft`.

Reply:
I'd prefer to be explicit. Additionally `check-manifest` is run as part of the lint `tox` environment, so GHA will tell us if someone forgets to add a file with a new extension to the Manifest.